### DISCUSSION TITLE
Galleria fullscreen with thumbnails

### DISCUSSION THREAD

**Discussion by BAZDemkivDev** [primefaces/community#1158] [PrimeVue]
Hi, I have some trouble with the Galleria component that has fullscreen property enabled with thumbnails.
Screenshot:
![Знімок екрана 2024-02-09 о 11 14 48](https://github.com/primefaces/community/assets/134921539/e1b098c7-cb81-40d1-bc33-3c6ef4a1b63f)
The thumbnails have overflow auto, visible scrolls and the photos in the thumbnails are very close to each other and have weird width.
Also, when navigating via thumbnail navigation, the items sometimes disappear, as seen on these screenshots:
![Знімок екрана 2024-02-09 о 11 15 05](https://github.com/primefaces/community/assets/134921539/e878e6d4-a006-4b0d-8529-97c81d2ca181)
![Знімок екрана 2024-02-09 о 11 15 24](https://github.com/primefaces/community/assets/134921539/4aae7f6a-284a-4eb8-9e17-3ec365bc77bf)
I expected it to be more like the original without the fullscreen property:
![Знімок екрана 2024-02-09 о 11 15 47](https://github.com/primefaces/community/assets/134921539/e8cf2a54-cd4c-4b52-b671-634a8a2b6fd2)
When discussing this with other developers, most of them said that older versions of PrimeVue didn't have this problem, but I couldn't find a working version of the component in older versions. Right now I'm using version 3.48.1, the previous versions that I've tried are: 3.47.0, 3.46.0 and 3.45.0. 
The screenshots provided are all from PrimeVue documentation: https://primevue.org/galleria/#fullscreenwiththumbnail
Looking forward to your reply!

**Comment by tugcekucukoglu** [comment]
Thanks for your attention.

Could you create an issue? It seems there is a bug here.

**Reply by BAZDemkivDev** [reply]
I created two issues:
https://github.com/primefaces/community/issues/1161
https://github.com/primefaces/primevue/issues/5267
